### PR TITLE
Show probe operating system

### PR DESCRIPTION
**Pull request by ceci21**
Implements #514 

Screen Shot 2023-02-07 at 10 34 59 AM
Screen Shot 2023-02-07 at 10 35 21 AM
Screen Shot 2023-02-07 at 10 35 35 AM

### CODE REVIEW

File: apps/probe/src/components/probe-list/probe-list.tsx [hunk 1]
```diff
@@ -23,6 +24,18 @@ import {
 } from "../../hooks/use-probes";
 import styles from "./probe-list.module.css";
 
+const getOsVariant = (dos: string) => {
```
Comment (kenrick) [inline apps/probe/src/components/probe-list/probe-list.tsx:27]: You can use the `Platform` type here it should come from the SDK

```suggestion
const getOsVariant = (dos: Platform) => {
```

File: apps/probe/src/components/probe-list/probe-list.tsx [hunk 2]
```diff
@@ -23,6 +24,18 @@ import {
...
+  if (!dos) return;
+
+  if (dos.includes("linux")) {
+    return "linux";
+  } else if (dos.includes("darwin")) {
+    return "darwin";
+  } else if (dos.includes("windows")) {
+    return "windows";
+  }
```
Comment (kenrick) [inline apps/probe/src/components/probe-list/probe-list.tsx:36]: The format for all the dos are pretty much the same so we can do some light parsing
```suggestion
return dos.split("-")[0]
```

**Review by kenrick — DISMISSED** [review]

File: apps/probe/src/components/probe-list/probe-list.tsx (hunk 2 above)
Comment (ceci21) [reply apps/probe/src/components/probe-list/probe-list.tsx:36]: Good to know, ty ty

**Review by kenrick — COMMENTED** [review]
Looks good!

**Pull request merged**